components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1097,6 +1097,17 @@ func (p *pg) DeleteRule(ctx context.Context, id string) error {
 		if err != nil {
 			return p.processError(err)
 		}
+		// Code is built around expectation that rules always have at least one condition,
```

Comment:
I'd prefer we fixed the other code. This is a bit too weird.

Reply:
I'm fine with that; I agree it definitely is kludgy.